cluster/common.sh

```diff
@@ -995,7 +995,12 @@ function generate-certs {
     mv "kubelet.pem" "pki/issued/kubelet.crt"
     rm -f "kubelet.csr"
 
-    ./easyrsa build-client-full kubecfg nopass
+    # Make a superuser client cert with subject "O=system:masters, CN=kubecfg"
+    ./easyrsa --dn-mode=org \
+      --req-cn=kubecfg --req-org=system:masters \
+      --req-c= --req-st= --req-city= --req-email= --req-ou= \
+      build-client-full kubecfg nopass
```

Comment:
I think upgrade looks like "keep existing ABAC policy file in place, unioned with RBAC policy", so all existing identities, tokens, certs, etc, keep working as-is.